src/endless/res/EndlessUsbTool.css

```diff
@@ -439,7 +438,7 @@ select:disabled {
 
 .PanelText {
 	width: 80%;
-	font-size: 15px;
+	font-size: 14px;
```

Comment:
This makes the “O”s in “OS” look terrible. Check out how the bottom is thinner than the top:

![screenshot from 2016-10-22 09-46-56](https://cloud.githubusercontent.com/assets/86760/19618248/8f1ac2ea-983c-11e6-95eb-9f9a5d6107e1.png)

For comparison, here's what this panel looks like at 15px:

![screenshot from 2016-10-22 10-25-06](https://cloud.githubusercontent.com/assets/86760/19618420/010f16a8-9842-11e6-9028-d03018a9f4be.png)

In a translation which wraps onto the third line, it still looks better than before – a little crowded but okay:

![screenshot from 2016-10-22 10-29-13](https://cloud.githubusercontent.com/assets/86760/19618455/8d9ddc12-9842-11e6-9f73-8c15432df019.png)